main/src/cgeo/geocaching/settings/CredentialsPreference.java

```diff
@@ -135,6 +136,9 @@ protected boolean isAuthorized() {
 
     @Override
     protected void revokeAuthorization() {
+        if (credentialsMapping == CredentialActivityMapping.GEOCACHING) {
+            Cookies.clearCookies();
```

Comment:
It shouldn't. Cookies store session information for sites not using OAuth. Clearing the cookies will go through the classical login phase (a few extra requests) when data is next requested, but no interaction will need to take place.